packages/jupyterlab-gridstack/src/editor/item/toolbar.tsx

```diff
@@ -0,0 +1,53 @@
+import { ReactWidget } from '@jupyterlab/apputils';
+
+import * as React from 'react';
+
+import { GridStackItemModel, ItemState } from './model';
+
+import { deleteIcon, pinIcon, unPinIcon } from '../icons';
+
+/**
+ * A React widget for items toolbar.
+ */
+export class GridStackItemToolbar extends ReactWidget {
+  constructor(model: GridStackItemModel) {
+    super();
+    this.addClass('grid-item-toolbar');
+    this._model = model;
+    this._model.stateChanged.connect(this._stateChanged);
+  }
+
+  dispose() {
+    this._model.stateChanged.disconnect(this._stateChanged);
+    super.dispose();
+  }
+
+  private _stateChanged = (
+    item: GridStackItemModel,
+    state: ItemState
+  ): void => {
+    this.update();
+  };
+
+  render(): JSX.Element {
+    return (
+      <>
+        {this._model.isLocked ? (
+          <div className="pin" onClick={() => this._model.unlock()}>
+            <unPinIcon.react height="16px" width="16px" />
```

Comment:
You should avoid hard coding styles (bad practice). Instead you should add a common class to all buttons and set the svg size using that class and CSS rules.